### PR TITLE
moves duplicated steps to background

### PR DESCRIPTION
**Pull request by individual-it**
## Description
moves steps into the background section

## Related Issue

#32040

## Motivation and Context
make the tests files shorter

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by paurakhsharma — APPROVED** [review]

File: tests/acceptance/features/apiSharing-v1/deleteShare.feature [hunk 1]
```diff
@@ -38,8 +34,7 @@ Feature: sharing
 		And as "user1" the folder "/sub" should not exist
 
 	Scenario: sharing subfolder of already shared folder, GET result is correct
-		Given user "user0" has been created
-		And user "user1" has been created
+		Given user "user1" has been created
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharing-v1/deleteShare.feature]: Can delete this line and make the next line ``Given``

File: tests/acceptance/features/apiSharing-v1/deleteShare.feature [hunk 2]
```diff
@@ -62,8 +57,7 @@ Feature: sharing
 		And file "/folder1/folder2" should be included as path in the response
 
 	Scenario: deleting a file out of a share as recipient creates a backup for the owner
-		Given user "user0" has been created
-		And user "user1" has been created
+		Given user "user1" has been created
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharing-v1/deleteShare.feature]: Can delete this line and make the next line ``Given``

File: tests/acceptance/features/apiSharing-v1/deleteShare.feature [hunk 3]
```diff
@@ -75,8 +69,7 @@ Feature: sharing
 		And as "user1" the file "/shared_file.txt" should exist in trash
 
 	Scenario: deleting a folder out of a share as recipient creates a backup for the owner
-		Given user "user0" has been created
-		And user "user1" has been created
+		Given user "user1" has been created
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharing-v1/deleteShare.feature]: Can delete this line and make the next line ``Given``

File: tests/acceptance/features/apiSharing-v1/deleteShare.feature [hunk 4]
```diff
@@ -91,8 +84,7 @@ Feature: sharing
 		And as "user1" the file "/sub/shared_file.txt" should exist in trash
 
 	Scenario: unshare from self
-		Given user "user0" has been created
-		And user "user1" has been created
+		Given user "user1" has been created
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharing-v1/deleteShare.feature]: Can delete this line and make the next line ``Given``

File: tests/acceptance/features/apiSharing-v1/deleteShare.feature [hunk 5]
```diff
@@ -105,8 +97,7 @@ Feature: sharing
 		And the etag of element "/PARENT" of user "user0" should not have changed
 
 	Scenario: sharee of a read-only share folder tries to delete the shared folder
-		Given user "user0" has been created
-		And user "user1" has been created
+		Given user "user1" has been created
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharing-v1/deleteShare.feature]: Can delete this line and make the next line ``Given``

File: tests/acceptance/features/apiSharing-v1/deleteShare.feature [hunk 6]
```diff
@@ -119,8 +110,7 @@ Feature: sharing
 		And as "user1" the file "/shared/shared_file.txt" should exist
 
 	Scenario: sharee of a upload-only shared folder tries to delete a file in the shared folder
-		Given user "user0" has been created
-		And user "user1" has been created
+		Given user "user1" has been created
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharing-v1/deleteShare.feature]: Can delete this line and make the next line ``Given``

File: tests/acceptance/features/apiSharing-v1/deleteShare.feature [hunk 7]
```diff
@@ -133,8 +123,7 @@ Feature: sharing
 		And as "user0" the file "/shared/shared_file.txt" should exist
 
 	Scenario: sharee of a upload-only share folder tries to delete his file in the folder
-		Given user "user0" has been created
-		And user "user1" has been created
+		Given user "user1" has been created
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharing-v1/deleteShare.feature]: Can delete this line and make the next line ``Given``

**Review by phil-davis — COMMENTED** [review]
See comments

**Comment by individual-it** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@PVince81 there is some codecov problem even though this is acceptance test feature file changes only.
Please merge.

**Comment by individual-it** [comment]
backport in  #32077

**Pull request merged**